### PR TITLE
Feature(theme): add nightmode react logic

### LINKED ISSUE #9953: [beta] Fix Night Mode

### PR DESCRIPTION
**Pull request by BerkeleyTrue**
We wait to load the user before applying the theme
as we will begin aggressively caching most of the react
app routes. This means we can not depend on user data before 
the page hit's the client

closes #9953

### CODE REVIEW

File: server/boot/settings.js [hunk 1]
```diff
@@ -49,8 +49,26 @@ export default function settingsController(app) {
...
+    return req.user.updateTheme('' + theme)
+      .then(
+        data => res.json(data),
+        next
+      );
+  }
+
   api.post(
     '/toggle-lockdown',
+    ifNoUser401,
```
Comment (BerkeleyTrue) [inline server/boot/settings.js]: I added this line here as it was missing, but it is not relevant to this feature.

**Comment by raisedadead** [comment]
@BerkeleyTrue Apologies, this needs a rebase, must have been due to me merging a pull request more recent than this one.

**Comment by BerkeleyTrue** [comment]
@raisedadead rebased

**Comment by raisedadead** [comment]
#### :sparkles: :sparkles: **LGTM.** :sparkles: :sparkles: 

Thanks @BerkeleyTrue !

**Pull request merged**